.github/workflows/ci.yml

```diff
@@ -0,0 +1,43 @@
+name: nf-core CI
+# This workflow runs the pipeline with the minimal test dataset to check that it completes without any syntax errors
+on:
+  push:
+    branches:
+      - dev
+  pull_request:
+  release:
+    types: [published]
+
+# Uncomment if we need an edge release of Nextflow again
+# env: NXF_EDGE: 1
+
+jobs:
+  test:
+    name: Run workflow tests
+    # Only run on push if this is the nf-core dev branch (merged PRs)
+    if: ${{ github.event_name != 'push' || (github.event_name == 'push' && github.repository == 'nf-core/rts') }}
```

Comment:
```suggestion
    if: ${{ github.event_name != 'push' || (github.event_name == 'push' && github.repository == 'qbic-pipelines/root-tissue-analysis') }}
```

Reply:
I think this was tackled, I can open an issue to make sure we clean up, but I am sure we will take care of that before release.